Edu_Platform/Forum/domain/models.py

```diff
@@ -1,3 +1,31 @@
 from django.db import models
+from django.utils import timezone
 
-# Create your models here.
+
+class ForumThread(models.Model):
+    course_id = models.IntegerField()
+    title = models.CharField(max_length=200)
+    content = models.TextField(null=True, blank=True)
+    created_date = models.DateTimeField(default=timezone.now)
+    
+    def __str__(self):
+        return f'{self.title} [Course ID: {self.course_id}]'
+
+
+class ThreadReply(models.Model):
+    thread = models.ForeignKey(
+        ForumThread,
+        on_delete=models.CASCADE,
+        related_name='replies'
+    )
+    user_id = models.IntegerField(null=True, blank=True)
+    content = models.TextField(null=True, blank=True)
+    created_date = models.DateTimeField(default=timezone.now)
+    
+    def __str__(self):
+        return f'Reply by User {self.user_id} on "{self.thread.title}"'
+
```

Comment:
Delete empty spaces

Reply:
@VitalikMatvieiev do you mean 'blank=True' or what ?